### DISCUSSION TITLE
modify page,it is not effective

### DISCUSSION THREAD

**Discussion by ganggangtian** [langgenius/dify#5621] [General]
I have modified the dify login page(dify\web\i18n\zh-Hans\login.ts), and then user(npm run dev) can run on this page by accessing http://localhost:3000/signin,the page has taken effect, but by accessing http://127.0.0.1/signin,the login page has not been updated with the latest page, and even restarting Docker is not he latest page.

**Comment by chinacqzgp** [comment]
It's also my question,who can answer ?

**Comment by crazywoola** [comment]
If you run things by using source code development, you will be navigate to `127.0.0.1:3000`.
But accessing by `http://127.0.0.1`, this is a url provided by docker compose ways. Which contains several configurations including nginx, and etc.

The are two different ways, if you want to make the second way succeed, you need to rebuilt the web image to make things take effect.

BTW, if you want to modify the logo and login method, you need to contact the bussiness@dify.ai.

**Reply by guailigirl** [reply]
How can I rebuilt the web image？I didn't find anything in the image named dify-web.
